data/apps.yml

```diff
@@ -1,119 +1,63 @@
+  summary:  "Mozilla Firefox Web Browser"
+  "flatpakref": "https://flathub.org/repo/appstream/org.mozilla.firefox.flatpakref"
+  icons:
+    "128":  "https://flathub.org/repo/appstream/x86_64/icons/128x128/org.mozilla.firefox.png"
+    "64":   "https://flathub.org/repo/appstream/x86_64/icons/64x64/org.mozilla.firefox.png"
 
-- name:     "Slack"
-  id:       "com.slack.Slack"
-  ref:      "app/com.slack.Slack/x86_64/stable"
-  summary:  "Chat with your team"
-  color:    "#C5BFB5"
-  "flatpakref": "https://flathub.org/repo/appstream/com.slack.Slack.flatpakref"
-  icons:
-    "64":   "https://lh3.googleusercontent.com/Dq-mZ5mmdE6aFPeD61DNlVTwYSI75UwHBYDq_BxBZOMSzCBnQ5OCC4-LjfP42tDlyw=w64"
-    
-- name:     "Inkscape"
-  id:       "org.inkscape.Inkscape"
-  ref:      "app/org.inkscape.Inkscape/x86_64/stable"
-  summary:  "Vector graphics editor"
-  color:    "#C5BFB5"
-  "flatpakref": "https://flathub.org/repo/appstream/org.inkscape.Inkscape.flatpakref"
-  icons:
-    "128":  "https://flathub.org/repo/appstream/x86_64/icons/128x128/org.inkscape.Inkscape.png"
-    "64":   "https://flathub.org/repo/appstream/x86_64/icons/64x64/org.inkscape.Inkscape.png"
+- name:     "OBS Studio"
+  id:       "com.obsproject.Studio"
+  ref:      "app/com.obsproject.Studio/x86_64/stable"
+  summary:  "Live streaming and video recording software"
+  "flatpakref": "https://flathub.org/repo/appstream/com.obsproject.Studio.flatpakref"
+  icons:
+    "128":  "https://flathub.org/repo/appstream/x86_64/icons/128x128/com.obsproject.Studio.png"
+    "64":   "https://flathub.org/repo/appstream/x86_64/icons/64x64/com.obsproject.Studio.png"
 
-- name:     "Spotify"
-  id:       "com.spotify.Client"
-  ref:      "app/com.spotify.Client/x86_64/stable"
-  summary:  "Online music streaming service"
-  color:    "#1DC558"
-  "flatpakref": "https://flathub.org/repo/appstream/com.spotify.Client.flatpakref"
-  icons:
-    "64":   "https://lh3.googleusercontent.com/UrY7BAZ-XfXGpfkeWg0zCCeo-7ras4DCoRalC_WXXWTK9q5b0Iw7B0YQMsVxZaNB7DM=w64"
-    
-- name:     "Steam"
-  id:       "com.valvesoftware.Steam"
-  ref:      "app/com.valvesoftware.Steam/x86_64/stable"
-  summary:  "Manage and play games distributed by Steam"
-  featured: "yes"
-  color:    "#7C8699"
-  "flatpakref": "https://flathub.org/repo/appstream/com.valvesoftware.Steam.flatpakref"
-  icons:
-    "128":  "https://flathub.org/repo/appstream/x86_64/icons/128x128/com.valvesoftware.Steam.png"
-    "64":   "https://flathub.org/repo/appstream/x86_64/icons/64x64/com.valvesoftware.Steam.png"
+- name:     "Builder"
+  id:       "org.gnome.Builder"
+  ref:      "app/org.gnome.Builder/x86_64/stable"
+  summary:  "An IDE for GNOME"
+  "flatpakref": "https://flathub.org/repo/appstream/org.gnome.Builder.flatpakref"
+  icons:
+    "128":  "https://flathub.org/repo/appstream/x86_64/icons/128x128/org.gnome.Builder.png"
+    "64":   "https://flathub.org/repo/appstream/x86_64/icons/64x64/org.gnome.Builder.png"
 
-- name:     "GIMP"
```

Comment:
Yeah, must not be merged until explained more thoroughly which criteria is violated.